### DISCUSSION TITLE
With disabled live history mode directory completion disappears

### DISCUSSION THREAD

**Discussion by underlow** [marlonrichert/zsh-autocomplete#485] [Q&A] [answered]
With disabled live history mode directory completion disappears
1. press CtrlR to disable live history
2. type `cd `
3. press Tab to see list of available directories
4. it shows but dissapears after releasing Tab key
5. it stays after pressing ArrowDown key while holding Tab pressed

It it a bug or a feature?

**Comment by marlonrichert** [comment, marked as answer]
Probably this bug: #483

**Reply by marlonrichert** [reply]
Bug has been fixed.
